### PR TITLE
corekit flag added

### PR DESCRIPTION
**Pull request by grvgoel81**
* corekit flag added
* Added Session Management
* Bump android sdk version

https://github.com/Web3Auth/web3auth-flutter-sdk/assets/44719313/3c5ae96c-431d-4372-afa9-fc74a372e7c5

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -50,6 +50,6 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1'
     implementation 'net.sourceforge.streamsupport:streamsupport-cfuture:1.7.4'
-    implementation 'org.torusresearch:web3auth-android-sdk:3.4.0'
+    implementation 'com.github.web3auth:web3auth-android-sdk:3.6.0'
```
Comment (chaitanyapotti) [inline android/build.gradle:53]: why is this changed to com.github?

File: android/src/main/kotlin/com/web3auth/flutter/web3auth_flutter/Web3AuthFlutterPlugin.kt [hunk 2]
```diff
@@ -133,6 +131,24 @@ class Web3AuthFlutterPlugin : FlutterPlugin, ActivityAware, MethodCallHandler,
                 }
                 return null
             }
+
+            "getPrivKey" -> {
+                val privKey = web3auth.getPrivkey()
```
Comment (chaitanyapotti) [inline android/src/main/kotlin/com/web3auth/flutter/web3auth_flutter/Web3AuthFlutterPlugin.kt:136]: do we handle exceptions ?

File: android/build.gradle (hunk 1 above)
Comment (grvgoel81) [reply android/build.gradle:53]: With org.torusresearch, its not working. When changed to com.github.web3auth, it starts working.

File: android/src/main/kotlin/com/web3auth/flutter/web3auth_flutter/Web3AuthFlutterPlugin.kt (hunk 2 above)
Comment (grvgoel81) [reply android/src/main/kotlin/com/web3auth/flutter/web3auth_flutter/Web3AuthFlutterPlugin.kt:136]: Internally native sdk's have exception handling mechanism.

**Review by chaitanyapotti — APPROVED** [review]
lgtm
